force-app/main/default/lwc/pharmanetHistory/pharmanetHistory.js

```diff
@@ -189,6 +189,11 @@ export default class PharmanetHistory extends LightningElement {
             dataArray.push(item);
           });
           this.data = dataArray;
+          this.data.sort((a, b) => {
+            let c = Date.parse(new Date(a.datedispensed));
```

Comment:
@RanadheerRG : Please do not use single variable literals. it should be more meaningful.